### PR TITLE
 usoc24: Add initial landing page

### PR DESCRIPTION
**Pull request by StefanJum**

### CODE REVIEW

File: content/hackathons/usw24/index.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: "Unikraft Summer Workshop 2024"
+description: Unikraft Summer Workshop 2024, July 1 - July 20, 2024
+startDate: 2024-07-01
+registerForm: https://forms.gle/6AMFoCoqJqKGdBgX6
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:5]:
```suggestion
registerForm: https://forms.gle/6AMFoCoqJqKGdBgX6
```

Should this be `registrationForm`?

File: content/hackathons/usw24/index.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+image: TODO
+---
+
+## Unikraft Summer Workshop 2024
+
+Unikraft Summer Workshop 2024 (*USW'24*) is a **free and virtual** unikernel and library operating systems workshop held by members of the Unikraft community.
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:11]:
```suggestion
Unikraft Summer Workshop 2024 (*USW'24*) is a **free and virtual** workshop held by members of the Unikraft community.
It focuses on cloud-native applications and on the unikernel technology that powers efficient and high performance cloud deployments.
```

File: content/hackathons/usw24/index.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+The three-week event holds a number of starter tutorials, workshops and lessons on how to get build Unikraft unikernels for production systems.
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:13]:
```suggestion
The three-week event holds a number of starter tutorials and workshops on how to configure, build, run, deploy and debug cloud applications using Unikraft.
```

File: content/hackathons/usw24/index.mdx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+The workshops will be hands-on and will take place for six days, between July 1 and July 12, 2024, 4pm-7pm CEST, in English, virtually via [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:15]:
```suggestion
There will be 6 sessions taking place in the first two weeks (between July 1 and July 12, 2024).
Each session takes place for 3 hours, in the 4pm-7pm CEST, in English.
Sessions will consist of talks and demos delivered by members of the Unikraft community, followed by practical tutorials that you will work on with support and supervision.
Sessions take place on [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
```

File: content/hackathons/usw24/index.mdx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+Topics include building unikernels, benchmarking, debugging, porting applications, virtualization and platform specifics.
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:16]:
```suggestion
Topics include building unikernels, benchmarking, debugging, porting applications, virtualization and platform specifics.
The first 3 sessions (in the first week) will focus on using KraftKit, Unikraft's compation tool, to manage cloud applications.
The next 3 sessions (in the second week) will focus on the internals of Unikraft: the build system, native configuration options, application porting.
```

File: content/hackathons/usw24/index.mdx [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+The workshops will be followed by a week of working on the final project.
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:17]:
```suggestion
The two weeks with sessions will be followed by a week of working on the final project.
You will work on the project in teams of 2-3 people.
```

File: content/hackathons/usw24/index.mdx [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+We will have support sessions online to help with the project, and on Saturday, 20 July 2024, we will have the final hackathon, online, on the [Unikraft's Discord server](), and for those who can make it, live, at [University POLITEHNICA of Bucharest]().
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:18]:
```suggestion
We will have support sessions online to help with the project.
On Saturday, 20 July 2024, 9am-5pm CEST, we will have the final hackathon.
The hackathon will take place in hybrid format, in person, at the [The National University of Science and Technology POLITEHNICA of Bucharest](https://upb.ro/en/), and, online, [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
```

File: content/hackathons/usw24/index.mdx [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+### Registration
+
+If you're eager to learn more about operating systems and unikernels, to work on practical open source tasks, to expand your knowledge of low-level topics, you'll want to be part of USW'24.
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:22]:
```suggestion
If you're eager to learn more about efficient cloud computing and unikernel technology. to work on practical open source tasks and to expand your knowledge of cloud-native and low-level topics, you'll want to be part of USW'24.
```

File: content/hackathons/usw24/index.mdx [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+To take part in the event, please fill this [registration form](https://forms.gle/6AMFoCoqJqKGdBgX6).
+You will need to complete a set of challenges by TODO, that will get you accustomed to the environment you will be using during the sessions.
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:24]:
```suggestion
You will need to complete a set of challenges that will get you accustomed to the environment you will be using during the sessions.
You will need to submit the challenge solution on the [registration form](https://forms.gle/6AMFoCoqJqKGdBgX6) by Thursday, June 27, 2024, 8pm AOE.
```

File: content/hackathons/usw24/index.mdx [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+
+It's recommended you check these prerequisites before taking part in USW'23:
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:26]:
```suggestion
It's recommended you check these prerequisites before taking part in USW'24:
```

File: content/hackathons/usw24/index.mdx [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+
+* fair knowledge of Linux command-line interface
+* good knowledge of programming concepts and the C programming language
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:29]:
```suggestion
* good knowledge of programming concepts;
   knowledge of the C programming language is a plus
```

File: content/hackathons/usw24/index.mdx [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+* basic understanding of operating system concepts: processes, threads, virtual memory, filesystems, file descriptors
+* some exposure to the assembly language and computer architecture
+* fondness for software engineering, hacking, tinkering with software components
+
+### People
+
+USW'24 will be held by members of the Unikraft community including professors and students from [University POLITEHNICA of Bucharest](https://upb.ro/) and the comercial side of Unikraft, [`Unikraft.io`](https://unikraft.io/).
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:36]:
```suggestion
USW'24 will be held by members of the Unikraft community including professors and students from [The National University of Science and Technology POLITEHNICA of Bucharest](https://upb.ro/) and the comercial side of Unikraft, [`Unikraft.io`](https://unikraft.io/).
```

File: content/hackathons/usw24/index.mdx [hunk 13]
```diff
@@ -0,0 +1,85 @@
...
+| Tue, 02.07.2023 | 3:30pm-4pm<br/> 4pm-7pm| Opening Ceremony<br/> Session 01: Overview of Unikraft |
+| Thu, 04.07.2023 | 4pm-7pm | Session 02: Baby Steps |
+| Fri, 05.07.2023 | 4pm-7pm | Session 03: Behind the Scenes |
+| Tue, 09.07.2023 | 4pm-7pm | Session 04: Binary Compatibility |
+| Thu, 11.07.2023 | 4pm-7pm | Session 05: Debugging in Unikraft |
+| Fri, 12.07.2023 | 4pm-7pm | Session 06: Application Porting |
+| Tue, 16.07.2023 | 4pm-7pm | Support Session 01 |
+| Thu, 18.07.2023 | 4pm-7pm | Support Session 02 |
+| Fri, 19.07.2023 | 4pm-7pm | Support Session 03 |
+| Sat, 20.07.2023 | 9am-7pm | Final Hackathon |
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:59]:
```suggestion
| Sat, 20.07.2023 | 9am-5pm | Final Hackathon |
```

File: content/hackathons/usw24/index.mdx [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+
+### Registration Challenges
+
+#### First Challenge - Cloud Master
+
+Create a docker-compose.yml file that sets up a simple stack consisting of a database, a monitoring service, and a simple service that queries the database.
```
Comment (razvand) [inline content/hackathons/usw24/index.mdx:65]:
```suggestion
Create a `docker-compose.yml` file that sets up a [Compose](https://docs.docker.com/compose/) application consisting of a database, a monitoring service, and a service that queries the database.
```

File: content/hackathons/usw24/index.mdx (hunk 1 above)
Comment (razvand) [reply content/hackathons/usw24/index.mdx:5]: The registration form is not available.

They should submit their GitHub ID, affiliation, e-mail address, (equivalent) year of study and motivation for taking part in the hackathon.

File: content/hackathons/usw24/index.mdx (hunk 7 above)
Comment (razvand) [reply content/hackathons/usw24/index.mdx:18]: Participants will receive a participation diploma.
The first three teams will get special prizes.
All hackathon in-person participants will get a Unikraft T-shirt.

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @StefanJum. I added my comments.

File: content/hackathons/usw24/index.mdx (hunk 1 above)
Comment (StefanJum) [reply content/hackathons/usw24/index.mdx:5]: Fixed, it was already created, but I forgot to enable it.

Comment (StefanJum) [reply content/hackathons/usw24/index.mdx:5]: > Should this be registrationForm?

No, this is how the renderer likes it.

File: content/hackathons/usw24/index.mdx (hunk 7 above)
Comment (StefanJum) [reply content/hackathons/usw24/index.mdx:18]: > special prizes

Do we know / can we announce the top prizes? It might rise more attention.

**Comment by StefanJum** [comment]
Done @razvand, please recheck.

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request merged**